### PR TITLE
Add various stuff regarding elx-netbilling

### PR DESCRIPTION
**Pull request by SereneVirus**

### CODE REVIEW

File: templates/elx_netbilling_agent.service.j2 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=ELX Net Billing Agent
+After=network.target
+
+[Service]
+Type=simple
+User=root
+Group=root
+ExecStart=/usr/local/bin/elx-netbilling-agent
```
Comment (Brittlejf) [inline templates/elx_netbilling_agent.service.j2]: Maybe we can use the path variable here as well? 
```suggestion
ExecStart={{ elx_netbilling_path }}
```

**Review by jhindersson — COMMENTED** [review]
As the netbilling agent uses standard openrc environment variables we also need to install one for the billing and make sure to load it with the systemd service:
EnvironmentFile

**Review by Brittlejf — APPROVED** [review]
Looks good to me overall. 
We discussed changing to a proper service user rather than the admin user, but that can be a different commit/PR when we have tested that this works.

**Comment by SereneVirus** [comment]
Yup, we agreed to use the admin credentials for Test and set up a service user that only can access Gnocchi for Prod (and later on do the same for Test, but not as urgent).

**Pull request merged**